docs/low-code-spark/gems/custom/file-operation.md

```diff
@@ -61,6 +66,69 @@ def copy_file(spark: SparkSession):
                  "/dbfs/Prophecy/example/target/person.json")
 ```
 
+</TabItem>
+<TabItem value="py3" label="S3">
+
+```py
+def copy_file(spark: SparkSession):
+    import boto3
```

Comment:
Can shorten these codeblocks to only show relevant code, rather than the whole codegen?

We can mention the boilerplate code above all at once perhaps

Reply:
@nadeesh-garg , I don't think we can remove any thing but if you feel so then let's discuss